futureTry/src/main/scala/com/softwaremill/futuretry/package.scala

```diff
@@ -0,0 +1,29 @@
+package com.softwaremill
+
+import scala.concurrent.{Promise, ExecutionContext, Future}
+import scala.util.{Success, Failure, Try}
+
+package object futuretry {
+
+  implicit class FutureTry[T](val baseFuture: Future[T]) extends AnyVal {
+
+    /**
+      * Reifies the Try output of Futures. Useful mostly for interaction with unusual APIs.
```

Comment:
Because it's expected by Scala's Standard Library to operate on the Future's `Try` "under the hood", so anything that makes the `Try` is explicit is, in that sense, unusual.